src/inet.c

```diff
@@ -685,7 +685,6 @@ void pr_inet_close(pool *p, conn_t *c) {
 
   if (c->pool != NULL) {
     destroy_pool(c->pool);
-    c->pool = NULL;
```

Comment:
The `c` object here is itself allocated out of the `c->pool`, thus attempting to dereference the `c` pointer here, after the call to `destroy_pool()`, is itself a use-after-free case.